src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Product/ProductSearch/Facets/FacetTreeVueComponent.ts

```diff
@@ -111,7 +116,7 @@ module Orckestra.Composer {
                             :data-selected="node.IsSelected"
                             v-on:click="(event) => nodeсlicked(event, node.IsSelected)" 
                         />
-                        {{node.Title}} ({{node.Quantity}})
+                        {{getTitle(node.Title)}} ({{node.Quantity}})
```

Comment:
Something wrong with this fix as problem is still present and comes not from Tree Nodes, but from Tree Title which is rendered in 
~Orckestra.Composer.Website\App_Code\FacetHelpers.cshtml
@helper CategoryFactesTree(string rootTitle)

I did investigation how it works on Grocery, and probably it was fixed time ago there, so try next fixes
1) Edit FacetHelpers.cshtml, line # 210, replace with next
 <strong><span>@Html.Raw(rootTitle)</span> ({{CategoryFacetValuesTree.TotalCount}})</strong>
2) Edit razor ~\src\Orckestra.Composer.Website\App_Data\Razor\Composer\BrowsingCategories\Facets.cshtml , line#51  replace with (add HttpUtility.HtmlEncode to category name)
@FacetHelpers.CategoryFactesTree(HttpUtility.HtmlEncode(categoryBrowsingViewModel.CategoryName))